lib/events/filelog.go

```diff
@@ -508,10 +553,17 @@ func (l *FileLog) findInFile(path string, filter searchEventsFilter) ([]EventFie
 	}
 	defer lf.Close()
 
+	lineNo := 0
 	// for each line...
-	retval := []EventFields{}
+	var retval []EventFields
 	scanner := bufio.NewScanner(lf)
-	for lineNo := 0; scanner.Scan(); lineNo++ {
+
+	// If custom MaxScanTokenSize was used allocate a custom buffer with a new size.
+	if l.MaxScanTokenSize != bufio.MaxScanTokenSize {
+		buf := make([]byte, 0, l.MaxScanTokenSize)
+		scanner.Buffer(buf, l.MaxScanTokenSize)
+	}
+	for ; scanner.Scan(); lineNo++ {
```

Comment:
i think alternatively maybe we can use `bufio.Reader.ReadLine` and checks `isPrefix` to combine big text to avoid the ErrTooLong